internal/app/machined/pkg/system/runner/goroutine/goroutine.go

```diff
@@ -89,14 +88,7 @@ func (r *goroutineRunner) wrappedMain() (err error) {
 	//nolint:errcheck
 	defer w.Close()
 
-	var writer io.Writer
-	if r.runtime.Config().Debug() {
```

Comment:
would we lose the debug ability here?

Reply:
yep, like with udevd, but it shouldn't be too crazy...